[2024-08-19T17:19:04.189+0500] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-08-19T17:19:04.208+0500] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 3_models_dag.prepare_data scheduled__2024-08-18T01:00:00+00:00 [queued]>
[2024-08-19T17:19:04.215+0500] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 3_models_dag.prepare_data scheduled__2024-08-18T01:00:00+00:00 [queued]>
[2024-08-19T17:19:04.215+0500] {taskinstance.py:2856} INFO - Starting attempt 1 of 1
[2024-08-19T17:19:04.234+0500] {taskinstance.py:2879} INFO - Executing <Task(PythonOperator): prepare_data> on 2024-08-18 01:00:00+00:00
[2024-08-19T17:19:04.246+0500] {logging_mixin.py:190} WARNING - /home/nicolaspc/anaconda3/envs/mlops_env/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=8325) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-08-19T17:19:04.247+0500] {standard_task_runner.py:72} INFO - Started process 8370 to run task
[2024-08-19T17:19:04.246+0500] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', '3_models_dag', 'prepare_data', 'scheduled__2024-08-18T01:00:00+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/train_models-2.py', '--cfg-path', '/tmp/tmpoy0poy0c']
[2024-08-19T17:19:04.247+0500] {standard_task_runner.py:105} INFO - Job 11: Subtask prepare_data
[2024-08-19T17:19:04.302+0500] {task_command.py:467} INFO - Running <TaskInstance: 3_models_dag.prepare_data scheduled__2024-08-18T01:00:00+00:00 [running]> on host nicolaspc-HP-Pavilion-Laptop-15-eh1xxx
[2024-08-19T17:19:04.378+0500] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Nicolas Pal' AIRFLOW_CTX_DAG_ID='3_models_dag' AIRFLOW_CTX_TASK_ID='prepare_data' AIRFLOW_CTX_EXECUTION_DATE='2024-08-18T01:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-18T01:00:00+00:00'
[2024-08-19T17:19:04.378+0500] {taskinstance.py:731} INFO - ::endgroup::
[2024-08-19T17:19:04.402+0500] {python.py:240} INFO - Done. Returned value was: {'start_tiemstamp': '20240819 17:19'}
[2024-08-19T17:19:04.429+0500] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-08-19T17:19:04.429+0500] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=3_models_dag, task_id=prepare_data, run_id=scheduled__2024-08-18T01:00:00+00:00, execution_date=20240818T010000, start_date=20240819T121904, end_date=20240819T121904
[2024-08-19T17:19:04.460+0500] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-08-19T17:19:04.487+0500] {taskinstance.py:3891} INFO - 3 downstream tasks scheduled from follow-on schedule check
[2024-08-19T17:19:04.489+0500] {local_task_job_runner.py:240} INFO - ::endgroup::
